src/FakeItEasy/Configuration/StartConfiguration.cs

```diff
@@ -26,7 +26,7 @@ public IReturnValueArgumentValidationConfiguration<TMember> CallsTo<TMember>(Exp
         {
             Guard.AgainstNull(callSpecification, nameof(callSpecification));
 
-            var parsedCallExpression = this.expressionParser.Parse(callSpecification, this.manager.Object);
+            var parsedCallExpression = this.expressionParser.Parse(callSpecification, this.manager.Object!);

```

Comment:
This method is called from `Fake<T>`, which has a strong reference to the fake object (proxy). So there's no chance that the fake could have been collected, so FakeManager.Object won't return null.